references/regex.json

```diff
@@ -1,3 +1,3 @@
 {
-  "submitter_id": "^[A-Za-z0-9\\-\\._]{1,64}$"
+  "submitter_id": "\b(?!DO|SP|SA|TR|PD|FU)\b^[A-Za-z0-9\\-\\._]{1,64}$"
```

Comment:
I plugged this in a few tests and i don't think its case sensitive - i think we would want it to be case senesitive so that someone cant submit `fu_234323` for example.  This is the case sensitive version that i got working: 
`\b(?!(?i:DO)|(?i:SP)|(?i:SA)|(?i:TR)|(?i:PD)|(?i:FU))\b^[A-Za-z0-9\\-\\._]{1,64}`  (test examples: https://regex101.com/r/Lx5q8n/1) 

Im not sure if there is a more elegant way to do this.

Reply:
Thanks @rosibaj I updated the regex with the one you suggested.